### ISSUE TITLE
Card image on membership site page in backend

### ISSUE THREAD

**Issue by nichart** [Simplero/Roadmap#416]
Would make sense to show card images on: https://sitename.simplero.com/admin/sites

**Comment by calvincorreli** [comment]
I agree. I've thought of it several times myself. Not sure why that wasn't done in the first place, to be honest.
